src/olympia/abuse/cinder.py

```diff
@@ -202,3 +204,33 @@ def get_context(self):
                 cinder_user.get_relationship_data(self, 'amo_rating_author_of')
             ],
         }
+
+
+class CinderAddonByReviewers(CinderAddon):
```

Comment:
nit: Ignore if you think that's not necessary/consistent with what you had in mind

```suggestion
class CinderAddonHandledByReviewers(CinderAddon):
```

Or something like that ?

Reply:
hadn't really thought that deeply about the naming, so sure